package.json

```diff
@@ -80,6 +80,7 @@
     "substyle": "^5.1.0"
   },
   "peerDependencies": {
+    "prop-types": "^15.5.8",
```

Comment:
should be in `dependencies`. (see https://www.npmjs.com/package/prop-types#how-to-depend-on-this-package)